Source/Core/Core/HW/GCKeyboardEmu.cpp

```diff
@@ -225,21 +225,21 @@ void GCKeyboard::LoadDefaults(const ControllerInterface& ciface)
 #elif __APPLE__
   m_keys0x->SetControlExpression(0, "Home");
   m_keys0x->SetControlExpression(1, "End");
-  m_keys0x->SetControlExpression(2, "Page Up");
-  m_keys0x->SetControlExpression(3, "Page Down");
+  m_keys0x->SetControlExpression(2, "`Page Up`");
+  m_keys0x->SetControlExpression(3, "`Page Down`");
   m_keys0x->SetControlExpression(4, "");  // Scroll lock
 
-  m_keys2x->SetControlExpression(9, "-");
+  m_keys2x->SetControlExpression(9, "`-`");
   m_keys2x->SetControlExpression(10, "Paragraph");
   m_keys2x->SetControlExpression(11, "");  // Print Scr
   m_keys2x->SetControlExpression(12, "'");
   m_keys2x->SetControlExpression(13, "[");
```

Comment:
All of these symbols will produce the error message too. I think you should wrap those with back ticks too.
Anything that isn't alphanumeric (or underscore) (and also starting with a letter) Is going to display the "syntax error" message.

Reply:
I've fixed the remaining ones I think. If there are any left, they can be done later on by someone else I guess.

Reply:
But yeah when adding these inputs to the expression, they should automatically be wrapped around `` unless they are fully alphanumeric I guess. I will have a look into that but it would be a different PR anyway.

Reply:
Never mind that's already the way it works.